scifight_proj/urls.py

```diff
@@ -13,9 +13,15 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from scifight import urls as scifight_urls
+from django.conf.urls import url, include
 from django.contrib import admin
+from scifight.views import index
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    url(r'^manage/',                            admin.site.urls),
```

Comment:
The thing you're trying to do is called [*security trough obscurity*][1], and is a bullshit. Anyone deploying the site may change this line to anything they want (like `^iddqd/`) if they want, and keep that exact string secret from the world. Let's just have the Django's default in our repository.

[1]: https://en.wikipedia.org/wiki/Security_through_obscurity